charts/base-cluster/values.schema.json

```diff
@@ -1263,6 +1263,102 @@
       },
       "additionalProperties": false
     },
+    "weaveworks": {
+      "type": "object",
+      "description": "Values for the gitops-chart",
+      "properties": {
+        "enabled": {
+          "type": "boolean"
+        },
+        "adminUser": {
+          "type": "object",
+          "properties": {
+            "create": {
+              "type": "boolean"
+            },
+            "username": {
+              "type": "string"
+            },
+            "createSecret": {
+              "type": "boolean"
+            },
+            "createClusterRole": {
+              "type": "boolean"
+            },
+            "passwordHash": {
```

Comment:
This should be the plain password and `bcrypt`ed via helm

Reply:
unfortunately this is not supported by the chart. You can enter a password hash, which is generated by their cli tool :disappointed:

Reply:
It should be, why not?

Reply:
This is also not tested yet but I generated the passwordHash with the httpwd helm function because the chat expected a bcrypt hash as input

Reply:
I did some adjustsments after the test. However, now it works. Maybe I should move the calculation to the _helper.tpl. What do you think?